### DISCUSSION TITLE
GHG Data Year for Latest Model Version

### DISCUSSION THREAD

**Discussion by emehlhop** [USEPA/useeior#278] [Q&A] [answered]
Hello :) we're using some relationships from the USEEIO matrices applied to the EPA SEF, and I know that with EPA SEF v1.2 the GHG data year is now 2019. I understand that 2016 GHG data was used for USEEIOv2.0 (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9065037/), but is my assumption correct that since both EPA SEF and USEEIO matrices are run on useeior, that using a later version of useeior (I used v1.3.0 at the time) will produce the USEEIO matrices using 2019 GHG data as well?

**Comment by emehlhop** [comment]
I produced USEEIOv2.1_422 for reference

**Reply by WesIngwersen** [reply, marked as answer]
Good question. The model specifications or configuration files that we use to build models are not always stored within useeior - in fact we have migrated away from doing that starting with v1.1. Models  with more recent GHG tables are found in [Supply-chain-factors](https://github.com/usepa/supply-chain-factors)
